src/main/scala/rise/eqsat/Analysis.scala

```diff
@@ -294,13 +294,13 @@ object NoAnalysis extends SemiLatticeAnalysis with TypeAnalysis {
   override def requiredTypeAnalyses(): Set[TypeAnalysis] = Set()
 
   override def make(egraph: EGraph, enode: ENode, t: TypeId,
-                    analysisOf: EClassId => ()): () = ()
+                    analysisOf: EClassId => Unit): Unit = ()
```

Comment:
```scala
object NoAnalysis extends SemiLatticeAnalysis with TypeAnalysis {
  type Data = ()
  type NatData = ()
  type TypeData = ()
```
Should these become `Unit` as well??

Reply:
Yes, I guess so. Everywhere where in the type `()` is used to mean Unit.
It's hard to search semantically like this in the codebase.

Could you make this update in the PR?

Reply:
@michel-steuwer should be good to merge now

Reply:
Great. Thanks!